res/shaders/structs.hlsl

```diff
@@ -38,7 +39,7 @@ struct SpotLight
     float near_plane;
     float3 specular;
 
-    float4x4 projection_view_model;
+    float4x4 projection_view;
```

Comment:
This whole thing seems like a fairly big change as well, was the previous calculation wrong?

Reply:
Projection_view_model is pretty much useless when transferring to deferred shading.